integrations/extensions/starter-kits/ibm-rpa/README.md

```diff
@@ -0,0 +1,107 @@
+
+One important caveate about this starter kit is that it does not contain the backend RPA distribution needed to complete an end-to-end demo. The purpose of the starter kit is to provide template assets needed to start building the integration but practitioners may need help from the IBM RPA client engineering team to build a working demo. IBM employees can put in a request for help through the Customer Care Jumpstart program. Please check the [intake process here through ISC](https://ibm.box.com/s/m715465sjeai35dz58rdp72t35z4z58m).
+
+## Starter Kit Overview
+
+Here is a list of the components included with this starter kit along with details on the purpose of each file:
+
+| File Name | Details |
+|-----------|---------|
+|cfInsert.js| Cloud function template used to insert an RPA item (e.g. job) into a queue to be executed. |
+|cfGetStatus.js| Cloud function template used to query RPA for item status. This is needed because RPA jobs can take longer than 30 seconds to complete which means you most poll this cloud function for the jobs completion status. |
+|RPAAsynchExtensionAPISpec.json| OpenAPI spec template that needs to be imported into Watson Assistant in order to integrate with the two cloud functions.|
+|RPAWA Integration.mp4| Very short video showing what this integration looks like when using the Watson Assistant preview panel.|
+|WA-IBM-RPA-Tester-action.json| Action skill sample that calls out to the RPA cloud functions using the above OpenAPI spec. |
+
+Note that the above templates will need to be modified according to the Integration Steps outlined below.
+
+## Integration Setup
+
+### Prerequisites
+
+To complete an end-to-end setup of this integration between Watson Assistant and IBM RPA you will need to provision accounts for all of the following:
+
+ - Watson Assistant (a trial account)
+ - IBM RPA
+ - IBM Cloud Functions (or some equivalent)
+
+### Setup RPA
+
+Go [here](https://www.ibm.com/docs/en/rpa/21.0?topic=automation-deploying-scripts) first for details on building and deploying RPA scripts.
+
+These instructions outline the basic steps: 
+
+1. Add credentials to your local machine so it can be unlocked.
+2. Build your script and publish (note the input and output variables as these will be used through out the setup of this integration).
+3. Define a system queue to use on the server.
+4. Workflow -> Processes -> Create Process
+    - Ensure variables match script (step 2) variables.
+
+Now ensure you can call that script via the rest api (Asynch/Synch). Note that this will require credentials.
+
+### Setup Cloud Functions
+
+First get the necessary parameters needed by the cloud functions:
```

Comment:
I suggest you add here a link to IBM cloud functions

Reply:
done